jobs/build/advisories/Jenkinsfile

```diff
@@ -94,6 +94,8 @@ node {
                 buildlib = load("pipeline-scripts/buildlib.groovy")
                 buildlib.kinit()
             }
+            
+            sshagent(["openshift-bot"]) {
```

Comment:
I started out being bothered that the contents didn't get indented. But there are a lot of contents, so it kind of made sense... but then, looking at the contents, I realized it's really only the interactions with git/github that require this credential. Just the bit under `stage("commit new advisories to ocp-build-data")` -- actually, just the few lines that actually run the git shell command. Could you move this down there please?